### PR TITLE
Issue management

### PR DESCRIPTION
**Pull request by jacqueswho**
### Reasons for making this change

These workflows will add 'need triage' label to all un labelled issues. By default an issue will have a label needs triage. This indicates that its a new issue and needs to be further investigated to decide further action and labels. If an issue has a label the 'needs triage' label will be removed

### Checklist
 I've added
* [x] **I'm adding or updating code**
* [x] **I'm adding a new feature**

### CODE REVIEW

**Comment by jacqueswho** [comment]
@epicfaace once this is merged, the workflow should start to mark all issues with label need triage. so this list can be sorted with correct labels

**Comment by jacqueswho** [comment]
@epicfaace can this be approved?

**Comment by jacqueswho** [comment]
@epicfaace is there something I must do?

**Comment by jacqueswho** [comment]
@epicfaace @heath-freenome included stale issue clean up for longer than 2 years

File: .github/workflows/clean_stale_issues.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Close inactive issues
```
Comment (heath-freenome) [inline .github/workflows/clean_stale_issues.yml:1]: I think that @epicfaace added a stale issue bot already?

**Review by heath-freenome — APPROVED** [review]

**Pull request merged**